eslint.config.js

```diff
@@ -72,87 +58,19 @@ const config = [
                     fixStyle: "separate-type-imports",
                 },
             ],
-            "@typescript-eslint/consistent-type-exports": [
-                "error",
-                {
-                    fixMixedExportsWithInlineTypeSpecifier: false,
-                },
-            ],
+            "@typescript-eslint/consistent-type-exports": "error",
             "@typescript-eslint/no-import-type-side-effects": "error",
             "@typescript-eslint/no-unused-vars": [
                 "error",
                 {
                     ignoreRestSiblings: true,
                 },
             ],
-            "comma-dangle": ["error", "always-multiline"],
-            "comma-spacing": [
-                "error",
-                {
-                    before: false,
-                    after: true,
-                },
-            ],
-            "comma-style": ["error", "last"],
-            "eol-last": ["error", "always"],
             "eqeqeq": ["error", "always"],
-            "function-paren-newline": ["off"],
-            "indent": ["off"],
-            "jsx-a11y/accessible-emoji": ["off"],
-            "no-console": ["off"],
-            "no-mixed-spaces-and-tabs": ["error"],
-            "no-unused-vars": [
-                "error",
-                {
-                    ignoreRestSiblings: true,
-                },
-            ],
-            "object-curly-newline": [
-                "error",
-                {
-                    consistent: true,
-                },
-            ],
-            "object-curly-spacing": ["error", "never"],
-            "object-property-newline": [
-                "error",
-                {
-                    allowAllPropertiesOnSameLine: true,
-                },
-            ],
-            "prefer-const": ["error"],
-            "quote-props": ["error", "consistent-as-needed"],
-            "quotes": ["error", "double"],
-            "react/jsx-tag-spacing": [
-                "error",
-                {
-                    beforeSelfClosing: "always",
-                },
-            ],
-            "react/react-in-jsx-scope": ["off"],
```

Comment:
Since you extend from `react.configs.flat["jsx-runtime"]`, I don't think the rule will give you false reports anymore.